### PR TITLE
Release v0.9.0

### PR DESCRIPTION
**Pull request by medzin**
Release v0.9.0

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.8.3-SNAPSHOT
+dev-master
```
Comment (AlfredBroda) [inline VERSION]: Why not 0.9.1-SNAPSHOT?

Comment (medzin) [reply VERSION]: I do not see the point of guessing the next version number.

Comment (AlfredBroda) [reply VERSION]: Me neither, actually. Where did dev-master come from?

Comment (medzin) [reply VERSION]: It was random - it sounds cool 😄

**Pull request merged**